tests/test_domain_limiter.py

```
@@ -0,0 +1,199 @@
+        t.join()
+
+    assert max_active <= 2
+    assert max_active >= 2
+
+
+def test_domain_limiter_leaves_other_domains_unthrottled():
+    limiter = DomainConcurrencyLimiter({"api.greenhouse.io": 1})
+    barrier = threading.Barrier(5)
+    failures = []
+
+    def worker():
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `worker`

Add return type annotation: `None`

(ANN202)